R/deployments.R

```diff
@@ -173,11 +174,12 @@ deploymentFile <- function(appPath, name, account, server) {
   file.path(accountDir, paste0(name, ".dcf"))
 }
 
-deploymentRecord <- function(name, account, server, appId, bundleId, url, when,
-                             metadata = list()) {
+deploymentRecord <- function(name, title, account, server, appId, bundleId, url,
+                             when, metadata = list()) {
   # compose the standard set of fields and append any requested
   as.data.frame(c(
       list(name = name,
+           title = if (is.null(title)) "" else title,
```

Comment:
Good question; it doesn't. We exclude the title from the API request entirely if it's not present:

https://github.com/rstudio/rsconnect/blob/691585f43aef947284789da50a95609eb9981922/R/connect.R#L95-L97